### DISCUSSION TITLE
Logging In

### DISCUSSION THREAD

**Discussion by idiglove** [Automattic/wp-calypso#53075] [Q&A] [answered]
Hi I'm new here
I have got it running locally. But logging in with Gmail SSO, doesnt work. It doesnt say I'm logged in.
Logging in with email, sends a verification email but the link there points me to `https://wordpress.com/log-in/link/use`

Any help will be good, thanks :)

**Comment by tyxla** [comment, marked as answer]
Howdy! Could you please open an issue with thorough step-by-step instructions on how to reproduce the problem? Thanks in advance!
